### PR TITLE
added Profile feature

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by anandateertha-dev** [comment]
Great work @HarshithaHShetty

**Comment by anandateertha-dev** [comment]
@HarshithaHShetty  The import statement for the Profile component is missing in App.jsx. Please add it and commit it.

**Comment by ghost** [comment]
I have committed the changes @anandateertha-dev

**Review by anandateertha-dev — CHANGES_REQUESTED** [review]
@HarshithaHShetty Please delete all the console log statements.

File: src/Components/NavBarComp/MainNavBar.jsx [hunk 1]
```diff
@@ -30,19 +30,21 @@ const MainNavBar = () => {
               headers,
             });
             setAvatarUrl(avatarResponse.data.avatar);
-            //console.log(avatarUrl);
```
Comment (anandateertha-dev) [inline src/Components/NavBarComp/MainNavBar.jsx]: Remove this console log

**Pull request merged**